docs/4_BATCH_WIND_TUNNELS.md

```diff
@@ -0,0 +1,10 @@
+### Batch wind tunnel examples
```

Comment:
Open a task to iterate on this. I think for a first run it is okay, but the purpose is to tell the story of this generation. I think we can do a first passage together and then Maya will put things in place, if @sergiomsantos agrees.

Reply:
[Here you go](https://github.com/inductiva/tasks/issues/40)